hello-strongbox-nuget-mono/README.md

```diff
@@ -0,0 +1,76 @@
+This is an example of how to use the Strongbox artifact repository manager with NuGet.
+
+## Pre-requisites
+The following is a minimal configuration in order to be able to use this tutorial:
+
+* `Linux` OS
```

Comment:
You right about Mono but I am not sure about all commands from this tutorial that they can run on Windows as is.